Purpose: Fix typo in fstab-decode(8) font escape
Author:  Bjarni Ingi Gislason <bjarniig@rhi.hi.is>
Fixes:   -
Status:  Not submitted upstream

---
--- a/man/fstab-decode.8
+++ b/man/fstab-decode.8
@@ -26,7 +26,7 @@ fstab-decode \- run a command with fstab-encoded arguments
 
 .SH DESCRIPTION
 .B fstab-decode
-decodes escapes in the specified \FIARGUMENT\fRs
+decodes escapes in the specified \fIARGUMENT\fRs
 and uses them to run \fICOMMAND\fR.
 The argument escaping uses the same rules as path escaping in
 \fB/etc/fstab\fR,
